feat(ranger): add action for extension-request (#2967)
cdr/code-server: docs/FAQ.md

### Installing an extension

Extensions can be installed from the marketplace using the extensions sidebar in
code-server or from the command line:

```shell
code-server --install-extension <extension id>
# example: code-server --install-extension wesbos.theme-cobalt2
```

## How can I request a missing extension?

We are currently in the process of transitioning to [Open VSX](https://open-vsx.org/).
Once https://github.com/eclipse/openvsx/issues/249
is implemented, we can fully make this transition. Therefore, we are no longer
accepting new requests for extension requests.

Instead, we suggest one of the following:

- [Switch to Open VSX](#how-do-i-configure-the-marketplace-url) now
- Download and [install the extension manually](#installing-an-extension-manually)

## Installing an extension manually

If an extension is not available from the marketplace or does not work, you can
grab its VSIX from its GitHub releases or build it yourself.

Once you have downloaded the VSIX to the remote machine you can either:

- Run the `Extensions: Install from VSIX` command in the Command Palette.
- Use `code-server --install-extension <path to vsix>`

> If the extension exists on another marketplace, you can also download it from there using the same command. For instance, downloading off OpenVSX can be done like this:

Author: ```suggestion
You can also download extensions from the command line. For instance, downloading off OpenVSX can be done like this:
```

code-server --install-extension downloaded-ms-python.python.vsix
```

> ### Installing VSIX Extensions via the Command Line

Reviewer: I think this overlaps a little with the previous paragraph that lists how to install extensions on the CLI.

What if we move that into this heading, then create another heading for this content with something like "Installing OpenVSX extensions via the command line" for this content?

We could also mention they can switch early to OpenVSX by setting `SERVICE_URL` and `ITEM_URL` (we already have a section so maybe we just link it from here).

Reviewer: Actually maybe we should put this under "How can I request a missing extension?"

Author: Agreed.

Yes! Let's do that!

Reviewer: Or just remove that section.

Author: I opted to add it under "How can I request a missing extension" - I say we move remove the extension request section once we fully switch to OpenVSX

Reviewer: My thinking is that the flow from a user might look like:

"How do I install an extension?" 
"Oh, it's not available, how do I request it?"
"Ah, they aren't accepting requests, how do I install it then?"
"OK, I can either do it manually or download from Open VSX."

Author: Ahh yeah good point about that. Okay I'll make these changes then re-request a review! Thanks for adding the structure/flow too! That helps a ton!!

Author: Embarrassingly, I didn't know you could do it this way, hence the `curl` suggestion:
```
SERVICE_URL=https://open-vsx.org/vscode/gallery ITEM_URL=https://open-vsx.org/vscode/item code-server --install-extension stylelint.vscode-stylelint
```
